presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -1087,13 +1088,20 @@ public MetastoreOperationResult addConstraint(MetastoreContext metastoreContext,
                         tableConstraint.isRely(),
                         tableConstraint.isEnforced());
             }
-            else {
+            else if (tableConstraint instanceof UniqueConstraint) {
                 tableConstraint = new UniqueConstraint(Optional.of(randomUUID().toString()),
                         tableConstraint.getColumns(),
                         tableConstraint.isEnabled(),
                         tableConstraint.isRely(),
                         tableConstraint.isEnforced());
             }
+            else if (tableConstraint instanceof NotNullConstraint) {
+                tableConstraint = new NotNullConstraint(Optional.of(randomUUID().toString()),
+                        tableConstraint.getColumns());
+            }
+            else {
+                throw new PrestoException(NOT_SUPPORTED, "This connector can only handle Unique/Primary Key constraints at this time");
```

Comment:
nit : Fix the exeception message to include 'can only handle Primary, Unique key and Not Nulll constraints '